packages/create-cloudflare/templates/simple/ts/wrangler.toml

```diff
@@ -1,5 +1,5 @@
 name = "<TBD>"
-main = "src/worker.ts"
+main = "src/worker.js"
```

Comment:
This should be `.ts` since the template is TS, right?

Reply:
Yep, fixed!